devices/xiaomi.js

```diff
@@ -2409,6 +2573,37 @@ module.exports = [
             await endpoint.read('aqaraOpple', [0x040a], {manufacturerCode: 0x115f});
         },
     },
+    {
+        zigbeeModel: ['aqara.feeder.acn001'],
+        model: 'ZNCWWSQ01LM',
+        vendor: 'Xiaomi',
+        description: 'Aqara Pet Feeder C1',
+        fromZigbee: [fzLocal.aqara_feeder],
+        toZigbee: [tzLocal.aqara_feeder],
+        exposes: [
+            exposes.enum('feed', ea.STATE_SET, ['START']).withDescription('Start feeding'),
+            exposes.enum('feeding_source', ea.STATE, ['manual', 'remote']).withDescription('Feeding source'),
+            exposes.numeric('feeding_size', ea.STATE).withDescription('Feeding size'),
+            exposes.numeric('portions_per_day', ea.STATE).withDescription('Portions per day'),
+            exposes.numeric('weight_per_day', ea.STATE).withDescription('Weight per day').withUnit('g'),
+            exposes.binary('error', ea.STATE, true, false).withDescription('Something wrong with feeder'),
+            exposes.list('schedule', ea.STATE_SET, exposes.composite('dayTime', exposes.access.STATE_SET)
```

Comment:
Does the list expose work correctly now (also in the frontend?)

Reply:
yes, with minor remarks, @nurikk is aware :)